### ISSUE TITLE
Add labels

### ISSUE THREAD

**Issue by amihaiemil** [zerocracy/farm#485]
0crat should add labels to the issues. 1 with the estimation, 1 with the assignee and 1 with the type (PR, bug etc)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@amihaiemil but we have `status` command for that... Or you think we still need labels? Also, we add `scope` label now.

**Comment by amihaiemil** [comment]
I still liked the labels better, you could clearly see who was assigned, what type and what the estimation of the issue was. 

Status should be used to check on 0crat if it's not acting as expected, I think. The less interaction with the bot, the better; why bother it for such info when it could just add the labels once.

And actually, no label of assignee is needed, you can see it clearly if 0crat will also do a Github assignment. This means that no label will ever have to be updated; maybe just add label ``x2``, ``x3`` etc, if the task gets a budget boost.

**Comment by g4s8** [comment]
@amihaiemil I don't think that we need a label with estimation, because all tasks are estimated as 30min, and I don't understand assignee label: 0crat is changing Github ticket assignee, you can filter by assignee by clicking on it in ticket list,  or didn't get you? But boost labels may have a sense, because we don't have Github native attributes for that.

**Comment by 0crat** [comment]
Job [#485](https://github.com/zerocracy/farm/issues/485) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)
